h2/src/main/org/h2/command/dml/Select.java

```diff
@@ -72,7 +72,7 @@
     /**
      * The main (top) table filter.
      */
-    TableFilter topTableFilter;
+    private TableFilter topTableFilter;
```

Comment:
Default visibility was correct here. `private` is enough for Java, but not for JVM, compiler creates an unnecessary synthetic method to resolve this problem.

The same in fields below.

Reply:
I'm OK with it for production code. Drop at all places in production code.